README.md

```diff
@@ -385,6 +385,23 @@ main().catch(console.error);
 
 A complete example can be found in [`samples/idtokens-iap.js`](https://github.com/googleapis/google-auth-library-nodejs/blob/master/samples/idtokens-iap.js).
 
+To explicitly acquire the ID token required to make HTTP requests to protected Cloud Run services, you can use the `getRequestMetadataAsync` method.
```

Comment:
I am curious what the use-case is for acquiring the ID token directly. The intention of the ID Token client is to make requests to protected Cloud Run services